content/applications/general/developer_mode.rst

```diff
@@ -33,10 +33,22 @@ Activate through a browser extension
    :align: center
    :alt: View of odoo’s debug icon in a chrome’s toolbar
 
+Activate through the command palette
+====================================
+
+The command palette tool has a command to activate the debug mode: open it with
+the keyboard shortcut `ctrl+k`, then type `debug`: a command will show up to
+activate the debug mode. 
+
+.. image:: command_palette.png
```

Comment:
Please move the file inside a new folder (if it doesn't exist yet) `developer_mode/`. We try to progressively move toward [this file structure](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files) and will someday script a migration of all media files. Until then, let's already adopt the new way.